api/envoy/service/status/v3/csds.proto

```diff
@@ -87,20 +69,14 @@ message ClientStatusRequest {
 }
 
 // Detailed config (per xDS) with status.
-// [#next-free-field: 8]
+// [#next-free-field: 7]
 message PerXdsConfig {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.service.status.v2.PerXdsConfig";
 
   // Config status generated by management servers. Will not be present if the
   // CSDS server is an xDS client.
-  ConfigStatus status = 1 [(udpa.annotations.field_migrate).oneof_promotion = "status_config"];
-
-  // Client config status is populated by xDS clients. Will not be present if
-  // the CSDS server is an xDS server. No matter what the client config status
-  // is, xDS clients should always dump the most recent accepted xDS config.
-  ClientConfigStatus client_status = 7
-      [(udpa.annotations.field_migrate).oneof_promotion = "status_config"];
+  ConfigStatus status = 1;
```

Comment:
This looks like a breaking API change.

Reply:
CSDS wasn't yet implemented by Envoy. I believe xDS management servers also haven't picked update this change, since it's meant for xDS clients. CC @fuqianggao.

Reply:
I'd want other @envoyproxy/api-shepherds to weight in on this one, since it's technically a breaking change to a stable API. We'd want to be really sure we have 100% global visibility here.

Reply:
I don't know of any clients that have actually implemented CSDS yet, so I suspect this is fine.  But it also doesn't hurt us to leave this here and mark it as deprecated.  I'm okay with either option.

Reply:
Leaving it in `csds.proto`, but marking it as deprecated.

Thanks for standing to keep backward compatibility.